src/go/rpk/pkg/cli/cmd/redpanda/admin/brokers/brokers.go

```diff
@@ -54,10 +54,26 @@ func newListCommand(fs afero.Fs) *cobra.Command {
 			bs, err := cl.Brokers()
 			out.MaybeDie(err, "unable to request brokers: %v", err)
 
-			tw := out.NewTable("Node ID", "Num Cores", "Membership Status")
+			headers := []string{"Node-ID", "Num-Cores", "Membership-Status"}
+
+			args := func(b *admin.Broker) []interface{} {
+				ret := []interface{}{b.NodeID, b.NumCores, b.MembershipStatus}
+				return ret
+			}
+			for _, b := range bs {
+				if b.IsAlive != nil {
+					headers = append(headers, "Is-Alive")
```

Comment:
We're inside a loop over the brokers so isn't this going to append an extra Is-Alive column for each broker, instead of just one column that is populated for each broker?

Reply:
The break just below quits this loop, the bit of logic in here is to ensure the column exists and to change the args func to account for the new column

Reply:
Silly me, that was a pre-coffee code review.  Never mind.